knock/src/core/ui/Reaction/Reaction.tsx

```diff
@@ -0,0 +1,55 @@
+import BeforeLike from '../../assets/icon/Before-like.svg';
+import BeforeHate from '../../assets/icon/Before-hate.svg';
+import AfterLike from '../../assets/icon/After-like.svg';
+import AfterHate from '../../assets/icon/After-hate.svg';
+import styles from '../../styles/Reaction.module.scss';
+import { useState } from 'react';
+
+interface ReactionProps {}
+
+const Reaction = ({}: ReactionProps) => {
+  const [likeCount, setLikeCount] = useState(0);
+  const [liked, setLiked] = useState(false);
+  const [unLiked, setUnLiked] = useState(false);
+
+  const handleLikeClick = () => {
+    const count = likeCount + 1;
+    setLikeCount(count);
```

Comment:
변수 선언 없이 사용할 수 있어서 이렇게 사용하는 것도 좋더라구요! 😄
```suggestion
    setLikeCount((prev) => prev+1);
```